Akash/akash-starter/docker-compose.yml

```diff
@@ -34,8 +34,9 @@ services:
     command:
       - -f=/app
       - --db-schema=app
-      - --disable-historical=false
+      - --workers=4
       - --batch-size=30
+      - # --unsafe=true #use this if you need to make network requests
```

Comment:
I think we can elaborate on this a lot more. I just gave a quick example in the issue.